### PR TITLE
Refactoring hdd scenario

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Closes #265

### CODE REVIEW

File: scenarios/hdd_monitoring/resources/vm.tf [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+data "opentelekomcloud_images_image_v2" "current_image" {
+  name        = var.ecs_image
+  most_recent = true
+}
+
+# Create security group for instances
+resource "opentelekomcloud_compute_secgroup_v2" "hdd_group" {
+  description = "Allow external connections to ssh, http, and https ports"
```
Comment (lego963) [inline scenarios/hdd_monitoring/resources/vm.tf]: Remove please http and https in description

File: scenarios/hdd_monitoring/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,4 @@
+availability_zone = "eu-de-01"
```
Comment (lego963) [inline scenarios/hdd_monitoring/terraform.tfvars]: You don't use this variable remove pls and from variables.tf

**Review by lego963 — CHANGES_REQUESTED** [review]

**Review by Polina-Gubina — DISMISSED** [review]

File: scenarios/hdd_monitoring/infrastructure.tf [hunk 3]
```diff
@@ -0,0 +1,23 @@
+locals {
+  workspace_prefix = terraform.workspace == "default" ? "" : terraform.workspace
+  key_pair = {
+    public_key = var.public_key
+    key_name   = "${local.workspace_prefix}_kp_${var.scenario}"
+  }
+}
+
+module "nodes" {
+  source               = "./resources"
```
Comment (lego963) [inline scenarios/hdd_monitoring/infrastructure.tf]: format pls

File: scenarios/hdd_monitoring/resources/vm.tf (hunk 1 above)
Comment (anton-sidelnikov) [reply scenarios/hdd_monitoring/resources/vm.tf]: fixed

File: scenarios/hdd_monitoring/terraform.tfvars (hunk 2 above)
Comment (anton-sidelnikov) [reply scenarios/hdd_monitoring/terraform.tfvars]: fixed

**Review by lego963 — DISMISSED** [review]

File: scenarios/hdd_monitoring/infrastructure.tf (hunk 3 above)
Comment (anton-sidelnikov) [reply scenarios/hdd_monitoring/infrastructure.tf]: formatted

**Review by lego963 — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**